src/main/java/emissary/core/BaseDataObject.java

```diff
@@ -754,10 +757,14 @@ public String printMeta() {
 
     @Override
     public void addProcessingError(final String err) {
-        if (this.procError == null) {
-            this.procError = new StringBuilder();
+        if (err == null) {
+            this.procError = null;
```

Comment:
Why would an "add" method remove existing entries?

Reply:
> What suggestion do you have for resolving your example above?

Revert the `addProcessingError` change.  If your `copy` use case needs a `clearProcessingErrors` method, create it. Don't add unexpected new behavior to the existing method just to satisfy the new edge case.